addon/components/bs-form.hbs

```diff
@@ -1,6 +1,5 @@
 {{!-- template-lint-disable no-invalid-interactive --}}
 <form
-  role="form"
```

Comment:
Wondering why we had it in the first place. Don't find it in Bootstrap v3 or v4 docs.